Directory.Build.props

```diff
@@ -85,9 +98,11 @@
     <MicrosoftExtensionsLoggingVersion>6.*</MicrosoftExtensionsLoggingVersion>
     <SystemTextEncodingsWebVersion>6.*</SystemTextEncodingsWebVersion>
     <MicrosoftExtensionsConfigurationBinderVersion>6.*</MicrosoftExtensionsConfigurationBinderVersion>
+    <MicrosoftExtensionsDependencyInjectionVersion>6.*</MicrosoftExtensionsDependencyInjectionVersion>
+
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(TargetFramework)' == 'net472' Or '$(TargetFramework)' == 'net462' Or '$(TargetFramework)' == 'net5.0'">
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net472' Or '$(TargetFramework)' == 'net462'">
```

Comment:
I believe that 472 could be a 2.x like netstandard2.0.
Let's check what George's doc or R9 do